src/misc/passwords/PasswordGeneratorDialog.ts

```diff
@@ -73,15 +74,11 @@ class PasswordGeneratorDialog implements Component<PasswordGeneratorDialogAttrs>
 			m(".small.mt-xs", [
 				lang.get("passphraseGeneratorHelp_msg"),
 				" ",
-				m(
-					"a",
-					{
-						href: "https://tuta.com/faq#passphrase-generator",
-						target: "_blank",
-						rel: "nooopener noreferer",
```

Comment:
we've lost some `nooopener noreferer` in places, do we need them?

Reply:
We might actually. I mistakenly thought `target: "_blank"` automatically includes both `noreferrer` & `noopener`. `target: "_blank"`  [automatically includes `noopener`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener) on [all browsers except Android Webview](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#browser_compatibility). `noreferrer`  [automatically includes `noopener`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noreferrer) on all browsers. So I will add just `noreferrer` back where needed.